pom.xml

```diff
@@ -256,7 +256,8 @@
         <mina.version>2.0.6</mina.version>
         <mpegts-streamer.version>0.1.0_2</mpegts-streamer.version>
         <net.minidev.asm.version>1.0.2</net.minidev.asm.version>
-        <netty.version>4.1.32.Final</netty.version>
+        <netty.version>4.1.44.Final</netty.version>
```

Comment:
Instead of separating these out, can we just upgrade netty as a whole to 46 instead of just netty codec?